src/components/ui/SkuSelector/sku-selector.scss

```diff
@@ -49,7 +49,7 @@
       &:checked + span {
         border-color: var(--color-border-input-selected);
         border-width: var(--border-width-2);
-        outline: var(--border-width-3) solid var(--bg-selected-outline);
+        box-shadow: 0 0 0 var(--border-width-3) var(--bg-selected-outline);
```

Comment:
Nicee, I was about to tell you this difference and then after refresh the PR the magic happens hahaha


> It looks like `focus` and `hover` states' have different styles. You can play a little getting the [SkuSelector](https://github.com/vtex-sites/base.store/blob/main-styled/src/components/ui/SkuSelector/sku-selector.scss#L37-L47) as a reference.